schema/google/showcase/v1beta1/sequence.proto

```diff
@@ -217,14 +217,19 @@ message AttemptSequenceRequest {
     (google.api.resource_reference).type = "showcase.googleapis.com/Sequence",
     (google.api.field_behavior) = REQUIRED
   ];
+
 }
 
 message AttemptStreamingSequenceRequest {
   string name = 1 [
     (google.api.resource_reference).type = "showcase.googleapis.com/StreamingSequence",
     (google.api.field_behavior) = REQUIRED
   ];
-
+  
+  int32 fail_index = 2 [
```

Comment:
Can we add some comments to explain what this field is used for? Also based on the server logic, this looks more like a `last_failed_index` or `restart_index`?

Reply:
good call, I like `last_failed_index`